ScreenshotMachine/CameraHandler.cs

```diff
@@ -17,18 +17,18 @@ public static class CameraHandler
 
         public static Sprite LineSprite { get; set; }
 
-        private static IEnumerator MoveCamera(Vector3 movement, KeyCode keyPressed)
+        private static IEnumerator MoveCamera(Vector3 movement, PlayerAction action)
         {
             if (!_frozenCamera)
             {
                 GameCameras.instance.cameraController.FreezeInPlace();
                 _frozenCamera = true;
             }
 
-            while (Input.GetKey(keyPressed))
+            while (action.IsPressed)
             {
                 GameCameras.instance.cameraController.transform.position += movement;
-                yield return new WaitForSeconds(0.1f);
+                yield return new WaitForSeconds(0.005f);
```

Comment:
I'd prefer this stay mostly unchanged, if you want to add speed something like ```cs
yield return new WaitForSeconds(Input.GetKey(Shift) ? 0.005f : 0.1)
```
would be preferable as the precision is very much wanted when taking screenshots.